docs/building.md

```diff
@@ -222,27 +222,74 @@ cargo run --release --bin slint-viewer -- examples/printerdemo/ui/printerdemo.sl
 
 ## Generating the Documentation
 
-### Rust
+The Slint documentation consists of five parts:
 
-The documentation for the different crates is built using rustdoc.
+- The tutorials
+- The Rust API documentation
+- The C++ API documentation
+- The Node.js API documentation
+- The DSL documentation
 
-The language documentation has snippets in the .slint language which can be previewed by injecting
-html to the documentation with the `--html-in-header` rustdoc flag.
+### Prerequisites
 
-Use the following command line to build the documentation to include preview of the .slint files.
+- [mdbook](https://rust-lang.github.io/mdBook/guide/installation.html)
+- [pipenv](https://pipenv.pypa.io/en/latest/)
+- [Python](https://www.python.org/downloads/)
+- [Doxygen](https://www.doxygen.nl/download.html)
```

Comment:
Not important, but shouldn't these specify for what it is?
For example, if i'm only interrested in building the nodejs docs, i don't need doxygen.